components/Content/CaseStudies.jsx

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import qs from 'qs';
+import useSWR from 'swr';
+
+import { Spinner } from '../Spinner';
+import Article from './Article';
+
+const URL = `${process.env.NEXT_PUBLIC_API_URL}/api`;
+const subURL = 'blog-posts';
+const fetcher = (...args) => fetch(...args).then((res) => res.json());
+
+export const CaseStudies = ({ limit }) => {
+  const params = {
+    sort: ['datePublished:desc'],
+    populate: '*',
+    'pagination[limit]': limit,
+    filters: {
+      id: { $gte: 88, $lte: 92 },
```

Comment:
Yes, there is no funnel when I checked the CMS and also the funnel for quarterly updates is unpublished. Otherwise, the plan was to use the funnel id to identify the blogs related to it, thought it would work in theory but can't test until the 2 funnels for quarterly updates and case studies are published :?

Reply:
Ok, this has been fixed now for both case studies and quarterly updates page! Now uses the funnel ids to filter out desired blog posts :D